src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -215,6 +215,7 @@ public function createTable(Table $table)
             $column = new Column();
             $column->setName('id')
                    ->setType('integer')
+                   ->setSigned(isset($options['signed']) ? $options['signed'] : false)
```

Comment:
This represents a backwards compatibility break as implemented. The way to do it without breaking BC would be to keep the current behavior (signed by default) and respect the `signed` option to create unsigned PKs.